metrics/prometheus/prometheus.go

```diff
@@ -1,196 +1,197 @@
-// (c) 2021, Ava Labs, Inc. All rights reserved.
+// (c) 2021-2025 Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
 package prometheus
 
 import (
+	"errors"
+	"fmt"
 	"sort"
 	"strings"
 
-	"github.com/ava-labs/subnet-evm/metrics"
-
 	"github.com/prometheus/client_golang/prometheus"
 
+	"github.com/ava-labs/libevm/metrics"
+
 	dto "github.com/prometheus/client_model/go"
 )
 
-var (
-	pv             = []float64{.5, .75, .95, .99, .999, .9999}
-	pvShortPercent = []float64{50, 95, 99}
-	pvShort        = []float64{.50, .95, .99}
-)
+func init() {
+	metrics.Enabled = true
+}
```

Comment:
Needed to keep the same behavior, since we modified geth to have `Enabled = true` at global scope instead of `false`.

Reply:
Could we add this to a different package's init, such as `plugin/evm` or make a new package like `plugin/evm/metrics`?

More ideal would be to set it in VM initialize but I'm not sure if that breaks too many tests.

Reply:
I suppose we could, tests affected are already changed to set enabled to true for the test duration. Also I hate global state, but oh well that's geth, no way around it (yet... 😈 )  🤷

Reply:
Done in initializeMetrics 😉